VSTSAgent/DSCResources/xVSTSAgent/xVSTSAgent.psm1

```diff
@@ -131,6 +131,10 @@ function Set-TargetResource {
         [System.String]
         $Work,
 
+        [parameter(Mandatory = $false)]
+        [System.String]
+        [string]$ProxyUrl,
```

Comment:
Sorry for missing this the first time, but we also need to add this flag to the Test-TargetResource function in this file (and we should test that the proxy setup is the same if we do find an agent, if possible). We also need to add this data to the .mof file next to these files.

Reply:
Done :)